<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapPlanning</class>
 <widget class="QDialog" name="MapPlanning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>1097</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="7">
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="deleteButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 4px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="addButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 4px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="updateTableButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>Update Table</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="executeButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="backButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="clearMapButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>Clear Map</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="clearTableButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>Clear Table</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="7">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>on_pushButton_7_clicked()</slot>
 </slots>
</ui>
